README.md

```diff
@@ -1,17 +1,55 @@
-# tailscale-client-go
+# tailscale.com/client/tailscale/v2
```

Comment:
out of curiosity, how are we pulling that off?  Are we planning to serve different go-get metadata at that URL?  (sorry, I'm behind on the latest client package naming discussions)